packages/block-library/src/navigation-link/editor.scss

```diff
@@ -1,4 +1,14 @@
 
+[data-type="core/navigation-link"] {
+
+	// Hide the draggable mover until functionality for positioning it on drag correctly is added
+	.block-editor-block-toolbar .components-toolbar.block-editor-block-mover {
+		.components-button.block-editor-block-mover__control-drag-handle {
```

Comment:
Does the selector need to be that specific?

Reply:
Unfortunately, yes :( 

I kept adding more classes to up the specificity until the change applied. Here's the selector it has to override:

```
.block-editor-block-contextual-toolbar:not([data-align="wide"]):not([data-align="full"]) .block-editor-block-mover:not(.is-horizontal) .block-editor-block-mover__control
```

Reply:
omg